src/test/java/fr/esgi/al5_2/Tayarim/ProprietaireServiceTest.java

```diff
@@ -26,262 +25,281 @@
+    Assertions.assertThat(proprietaireDTO.getNom()).isEqualTo(proprietaire.getNom());
+    Assertions.assertThat(proprietaireDTO.getPrenom()).isEqualTo(proprietaire.getPrenom());
+    Assertions.assertThat(proprietaireDTO.getEmail()).isEqualTo(proprietaire.getEmail());
+    Assertions.assertThat(proprietaireDTO.getNumTel()).isEqualTo(proprietaire.getNumTel());
+    Assertions.assertThat(proprietaireDTO.getDateInscription())
+        .isEqualTo(proprietaire.getDateInscription());
+
+  }
+
+  @Test
+  public void ProprietaireService_GetProprietaireByEmail_ReturnOneProprietaireDto() {
+    LocalDateTime localDateTime = LocalDateTime.now();
+
+    Proprietaire proprietaire = Proprietaire.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .motDePasse("password")
+        .dateInscription(localDateTime)
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaire.setLogements(null);
+    proprietaire.setId(1L);
+    when(proprietaireRepository.findFirstByEmail(proprietaire.getEmail())).thenReturn(
+        Optional.of(proprietaire));
+
+    ProprietaireDto proprietaireDTO = proprietaireService.getProprietaireByEmail(
+        proprietaire.getEmail());
+
+    Assertions.assertThat(proprietaireDTO).isNotNull();
+    Assertions.assertThat(proprietaireDTO.getId()).isEqualTo(proprietaire.getId());
+    Assertions.assertThat(proprietaireDTO.getNom()).isEqualTo(proprietaire.getNom());
+    Assertions.assertThat(proprietaireDTO.getPrenom()).isEqualTo(proprietaire.getPrenom());
+    Assertions.assertThat(proprietaireDTO.getEmail()).isEqualTo(proprietaire.getEmail());
+    Assertions.assertThat(proprietaireDTO.getNumTel()).isEqualTo(proprietaire.getNumTel());
+    Assertions.assertThat(proprietaireDTO.getDateInscription())
+        .isEqualTo(proprietaire.getDateInscription());
+
+  }
+
+  @Test
+  public void ProprietaireService_UpdateProprietaire_ReturnOneProprietaireDto() {
+    LocalDateTime localDateTime = LocalDateTime.now();
+    Long id = 1L;
+    ProprietaireCreationDto proprietaireCreationDTO = ProprietaireCreationDto.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .build();
+    Proprietaire proprietaire = Proprietaire.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .motDePasse("password")
+        .dateInscription(localDateTime)
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaire.setLogements(null);
+    proprietaire.setId(id);
+    when(proprietaireRepository.save(Mockito.any(Proprietaire.class))).thenReturn(proprietaire);
+    //ProprietaireDTO proprietaireDTO = proprietaireService.creerProprietaire(proprietaireCreationDTO);
+    ProprietaireUpdateDto proprietaireUpdateDTO = ProprietaireUpdateDto.builder()
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'proprietaireUpdateDTO' must contain no more than '1' consecutive capital letters.